test_scripts/API/SubtleAlertStyle/SubtleAlert/009_GetSystemCapabilities_no_app_specific_capabilities.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- User story: SubtleAlert cases
+-- Use case: SubtleAlert
+-- Item: Happy path (UI only)
+--
+-- Requirement summary:
+-- [SubtleAlert] Capabilities: getting the displayCapabilities with subtleAlert related params
+--
+-- Description:
+-- Mobile application sends valid GetSystemCapability request with DISPLAYS systemCapabilityType 
+-- and gets SUCCESS resultCode only if SDL received OnSystemCapabilityUpdated notification from HMI
+-- with the displayCapabilities for specific appID
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+
+-- Steps:
+-- App requests GetSystemCapability with DISPLAYS systemCapabilityType
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if displayCapabilities is available for specified appID
+-- SDL responds with (resultCode: DATA_NOT_AVAILABLE, success:false) to mobile application
+-- HMI sends OnSystemCapabilityUpdated with the displayCapabilities for specific appID
+-- SDL responds with (resultCode: SUCCESS, success:true) and transfers the displayCapabilities to mobile application
```

Comment:
Seems described sequence is not correct since SDL responds twice:
```
SDL responds with (resultCode: DATA_NOT_AVAILABLE, success:false) 
SDL responds with (resultCode: SUCCESS, success:true)
```